app/models/organization.rb

```diff
@@ -12,15 +12,14 @@ class Organization < ApplicationRecord
   acts_as_followable
 
   has_many :api_secrets, through: :users
-  has_many :articles
-  has_many :listings
-  has_many :collections
-  has_many :credits
-  has_many :display_ads
-  has_many :notifications
+  has_many :articles, dependent: :nullify
+  has_many :listings, dependent: :nullify
+  has_many :collections, dependent: :nullify
+  has_many :credits, dependent: :restrict_with_exception
+  has_many :display_ads, dependent: :destroy
+  has_many :notifications, dependent: :destroy
```

Comment:
Orgs notifications should go if the user goes. I don't think there's a case where a notification is the same object for a user and and org, correct?

Reply:
@Zhao-Andy is working on adding the ability to destroy organizations here https://github.com/forem/forem/pull/9357

See https://github.com/forem/forem/pull/9357/files#diff-33bef3083bae5b2e2bea316eab44c9fdR55 especially